dist/systemd/obs-api-setup.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description = Open Build Service API Setup: initialize and update api database
+BindsTo = obs-api-support.target
+
+[Service]
+Type = oneshot
+ExecStart = /usr/bin/sh -c "/usr/sbin/rcobsapisetup start"
```

Comment:
@coolo:

> these rc* links call systemctl - so either use systemctl directly on our own service - or require the service directly instead of wrapping sysvinit into systemd into bash

Not all of rc* links call systemctl. This one is `/usr/sbin/rcobsapisetup -> /etc/init.d/obsapisetup`.

Reply:
No, all rc* links call systemctl - see https://github.com/openSUSE/aaa_base/blob/master/files/etc/rc.status#L76